jcl/src/openj9.gpu/share/classes/com/ibm/gpu/Version.java

```diff
@@ -29,6 +29,13 @@
  */
 public class Version {
 
+	/**
+	 * Construct a new instance.
+	 */
+	public Version() {
```

Comment:
Is there a reason to have a public constructor?  I also wonder about making this final